### PR TITLE
Update license to lgplv3

### PR DESCRIPTION
**Pull request by jordy25519**
Also doughnut integration tests moved into the proper place

### CODE REVIEW

**Review by aliXsed — APPROVED** [review]
LGTM

File: runtime/tests/mock.rs [hunk 1]
```diff
@@ -15,7 +15,6 @@
 // along with CENNZnet.  If not, see <http://www.gnu.org/licenses/>.
 
 #![allow(dead_code)]
-#![cfg(test)]
```
Comment (cowboy-bebug) [inline runtime/tests/mock.rs:18]: I thought we needed this attribute to include it in cargo test.
I'm just double checking `cargo test -p cennznet-runtime` now

Comment (jordy25519) [reply runtime/tests/mock.rs:18]: It's not neccsesary for integration tests since they're in `/tests/` folder which cargo knows about

**Review by cowboy-bebug — APPROVED** [review]
LGTM thanks!

**Pull request merged**